apps/backend/src/modules/users/user.repository.ts

```diff
@@ -104,6 +106,12 @@ class UserRepository implements Repository {
 					notificationFrequency: user.userDetails.notificationFrequency,
 					passwordHash: user.passwordHash,
 					passwordSalt: user.passwordSalt,
+					taskPercentage:
+						(user.userTasks.filter(
+							(task) => task.status === TaskStatus.COMPLETED,
+						).length /
+							user.userTasks.length) *
+						ONE_HUNDRED,
```

Comment:
We shouldn't have this logic in repository. Repository should only be responsible for working with DB

Reply:
Should I put this on the frontend then just return all the task from backend?